### PR TITLE
Fix Half Light Machine Gun Ammo(IS)

### PR DESCRIPTION
**Pull request by binaryspica**
Fix for issue [#314 of MegaMekLab](https://github.com/MegaMek/megameklab/issues/314#issue-456358053)

Change setClanAdvancement to setISAdvancement and make it available for IS, not Clans, and also change the year. For now the year is same as Clans half Light Machine Gun Ammo, not IS Full Light Machine Gun Ammo.

I have fixed the line 5801~ to 5806 as well, making it resemble full Light Machine Gun Ammo(IS), but if it seems wrong please correct me.

### CODE REVIEW

**Comment by binaryspica** [comment]
And I have a question. Do we need to edit the line 5802(setAvailability) as well?

**Comment by neoancient** [comment]
See the weapon for the correct values: `megamek.common.weapons.mgs.ISLightMachineGun`

**Comment by binaryspica** [comment]
Thanks. I have checked it and fix the value as same as [ISLightMG.java](https://github.com/MegaMek/megamek/blob/master/megamek/src/megamek/common/weapons/mgs/ISLightMG.java).

File: megamek/src/megamek/common/AmmoType.java [hunk 1]
```diff
@@ -5740,7 +5740,7 @@ private static AmmoType createISLightMGAmmo() {
         .setIntroLevel(false)
         .setUnofficial(false)
         .setTechRating(RATING_B)
-        .setAvailability(RATING_X, RATING_X, RATING_C, RATING_B)
+        .setAvailability(RATING_X, RATING_C, RATING_C, RATING_B)
```
Comment (neoancient) [inline megamek/src/megamek/common/AmmoType.java]: In my previous answer I was giving a general answer about weapon and ammo tech advancement data and didn't look closely at the weapon availability codes and it turns out that there's a error in `ISLightMG`. The LMG did not exist in the SW era and the availability is XXCB.
Screenshot from 2019-06-14 16-33-39

**Comment by binaryspica** [comment]
Then we have to change ISLightMG as well. Thanks for the reference. And I am still confused since I am not familar to Github....

**Pull request merged**